SignalServiceKit/src/Messages/Interactions/TSMessage.m

```diff
@@ -245,14 +252,18 @@ - (NSString *)previewTextWithTransaction:(YapDatabaseReadTransaction *)transacti
             return NSLocalizedString(@"UNKNOWN_ATTACHMENT_LABEL", @"In Inbox view, last message label for thread with corrupted attachment.");
         }
     } else {
-        return self.body;
+        // TODO: We should do better here.
```

Comment:
Outside of a buggy sending client, does this happen?

In other words, should this be an OWSFail or at least a DDLogError

Reply:
Done.